test/core/update_taxon_assignments_test.py

```diff
@@ -0,0 +1,90 @@
+import os
+import time
+import unittest
+from configparser import ConfigParser
+from uuid import uuid4
+
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+from GenomeFileUtil.GenomeFileUtilServer import MethodContext
+from installed_clients.WorkspaceClient import Workspace as workspaceService
+
+# NOTE: These tests must run against https://ci.kbase.us
+_WORKSPACE_ID = 33192
```

Comment:
This is a public workspace on CI that I use for lots of tests... but we have to make sure it stays there. Not sure how to make this less brittle.

Reply:
if object 33 is a genome we could upload it here from fasta/gff or genbank files. Not sure if thats worth it

Reply:
That's true but it would take soooo long

Reply:
as long as its public I think its okay. maybe as a failsafe we store the genbank or fasta/gff in the test/data file here in case the workspace ever gets deleted